tests/influence/test_influences.py

```diff
@@ -76,52 +77,24 @@ def analytical_linear_influences(
     return result
 
 
-INFLUENCE_TEST_CONDITION_NUMBERS: List[int] = [3]
-INFLUENCE_TRAINING_SET_SIZE: List[int] = [50, 30]
-INFLUENCE_TEST_SET_SIZE: List[int] = [20]
-INFLUENCE_DIMENSIONS: List[Tuple[int, int]] = [
-    (10, 10),
-    (3, 20),
-]
-HESSIAN_REGULARIZATION: List[float] = [0, 1]
-
-
-test_cases = list(
-    itertools.product(
-        INFLUENCE_TRAINING_SET_SIZE,
-        INFLUENCE_TEST_SET_SIZE,
-        InfluenceType,
-        INFLUENCE_DIMENSIONS,
-        INFLUENCE_TEST_CONDITION_NUMBERS,
-        HESSIAN_REGULARIZATION,
-    )
-)
-
-
-def lmb_test_case_to_str(packed_i_test_case):
-    i, test_case = packed_i_test_case
-    return (
-        f"Problem #{i} of dimension {test_case[3]} with train size {test_case[0]}, "
-        f"test size {test_case[1]}, if_type {test_case[2]}, condition number {test_case[4]} and lam {test_case[5]}."
-    )
-
-
-test_case_ids = list(map(lmb_test_case_to_str, zip(range(len(test_cases)), test_cases)))
-
-
 @pytest.mark.torch
 @pytest.mark.parametrize(
-    "train_set_size,test_set_size,influence_type,problem_dimension,condition_number, hessian_reg",
-    test_cases,
-    ids=test_case_ids,
+    "influence_type",
+    InfluenceType,
+    ids=[ifl.value for ifl in InfluenceType],
+)
+@pytest.mark.parametrize(
+    "train_set_size",
+    [200],
+    ids=["train_set_size_200"],
 )
 def test_influence_linear_model(
```

Comment:
Probably better to split up in different test cases or using parameterization.

Reply:
I have refactored it a bit. Hopefully it is better.